### ISSUE TITLE
How to use predicted_variance in deepfloyd for textual inversion tuning?

### ISSUE THREAD

**Issue by hadaev8** [huggingface/diffusers#3307]
Im confused with 3 input and 6 output channels and not sure how to calculate loss

I checked pipeline code and example for textual inversion.

https://github.com/huggingface/diffusers/blob/eade4308dabc7f7ba75eab508d386b66b3764513/src/diffusers/pipelines/deepfloyd_if/pipeline_if_img2img.py#L927

Seems like channels reduced by scheduler?

**Comment by williamberman** [comment]
hey @hadaev8 yes very good point! We discussed this quite a bit last week. For finetuning, we're taking the approach of only training the predicted error on the simplified learning objective and then updating the scheduler configuration to ignore the predicted variance. It's worked on dream booth lora so far, hoping to test on textual inversion today as well :)

**Comment by hadaev8** [comment]
@williamberman 
Thanks for answer.
Any hint what param i need to change in scheduler config?

**Comment by williamberman** [comment]
`variance_type` :)

**Comment by williamberman** [comment]
going to resolve this for now as dreambooth training scripts with the non-variance training are merged :) feel free to re-open if needed

**Comment by hadaev8** [comment]
Okay i see, thanks.

**Comment by oss-roettger** [comment]
I have just published an Open Source version of Textual Inversion to incorporate your own objects, faces, logos or styles into
DeepFloyd IF: https://github.com/oss-roettger/T5-Textual-Inversion
The clear and short notebook runs within 24 GB CUDA memory. Feedback welcome!
Kind regards
Hans

**Comment by cfeng16** [comment]
Quick question, how to do "updating the scheduler configuration to ignore the predicted variance"? Thanks in advance.
